components/hub75_matrix_display/display.py

```diff
@@ -37,9 +37,65 @@
 I2SSPEED = "i2sspeed"
 LATCH_BLANKING = "latch_blanking"
 CLOCK_PHASE = "clock_phase"
+DOUBLE_BUFFER = "double_buffer"
 
 USE_CUSTOM_LIBRARY = "use_custom_library"
 
+VIRTUAL_MATRIX = "virtual_matrix"
```

Comment:
Personally, I would avoid the use of the "VM" acronym here for better readability.